test/suite/test_util01.py

```diff
@@ -161,13 +182,19 @@ def dump(self, usingapi, hexoutput):
                 expectout.write('table:' + self.tablename + '\n')
                 expectout.write('colgroups=,columns=,' + params + '\n')
                 expectout.write('Data\n')
-            for i in range(0, self.nentries):
-                key = self.get_key(i)
-                value = self.get_value(i)
-                cursor[key] = value
-                expectout.write(self.dumpstr(key, hexoutput))
-                expectout.write(self.dumpstr(value, hexoutput))
-        cursor.close()
+            if commit_timestamp and read_timestamp:
```

Comment:
Not sure if this is the best way to approach creating tests. Would love to know if this method is fine or not?